### ISSUE TITLE
Bug Report: Crash on copying to clipboard

### ISSUE THREAD

**Issue by MathiasMagnus** [microsoft/terminal#3007]
Copying text to the clipboard is extremely flaky.

# Environment

```none
[Environment]::OSVersion

Platform ServicePack Version      VersionString
-------- ----------- -------      -------------
 Win32NT             10.0.18362.0 Microsoft Windows NT 10.0.18362.0

Windows Terminal (Preview)
Version: 0.5.2681.0
```

# Steps to reproduce

Open up any console app and copy text via mouse clicks. [OneDrive link to desktop recording via Radeon Software.](https://1drv.ms/v/s!AmBiyz2ky0L5ru9MqJO4Kxu_cpYAqA?e=kTbumm)

# Expected behavior

Text be on clipboard and not crash.

# Actual behavior

Sometimes the app crashes after ~0.5 seconds when trying to copy to the clipboard (content is lost), even if copying is successful, closing the app clears the clipboard. Copying via Ctrl+C is more reliable, but Ctrl+C is reserved for `break`.

**Comment by Fractura** [comment]
This sounds close enough to the issues mentioned in https://github.com/microsoft/terminal/pull/2927 - so hopefully fixed with the next release.

**Comment by carlos-zamora** [comment]
Yeah. This'll be fixed by #2927. Thanks @Fractura for finding it.
